src/coreclr/vm/stackwalk.cpp

```diff
@@ -2882,6 +2877,15 @@ void StackFrameIterator::ProcessCurrentFrame(void)
                     SyncRegDisplayToCurrentContext(pRD);
                 }
             }
+            else if (InlinedCallFrame::FrameHasActiveCall(m_crawl.pFrame) && m_crawl.pFrame->PtrNextFrame() != FRAME_TOP && m_crawl.pFrame->PtrNextFrame()->GetFrameIdentifier() == FrameIdentifier::InterpreterFrame)
```

Comment:
m_Next can be NULL so PtrNextFrame can be NULL. But I assume in this case we know that it's non-null so it's okay to not have a null check before doing `->GetFrameIdentifier()` on it? I'm guessing an active call means that by definition we have some sort of managed frame - interpreted or native - above us that's performing the call?

Reply:
`m_Next` cannot be NULL. It can only be a valid pointer or the `FRAME_TOP`

Reply:
I see, when we set it to NULL in Pop and Remove we unlink ourselves first so that shouldn't be observable.